config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java

```diff
@@ -28,10 +29,26 @@
  */
 public final class ConfigurationFactory {
     private static final String REGISTRY_CONF = "registry.conf";
+    private static final String REGISTRY_CONF_PREFIX = "registry";
+    private static final String REGISTRY_CONF_SUFFIX = ".conf";
+    private static final String ENV_SYSTEM_KEY = "SEATA_CONFIG_ENV";
+    private static final String ENV_PROPERTY_KEY = "env";
     /**
      * The constant FILE_INSTANCE.
      */
-    public static final Configuration FILE_INSTANCE = new FileConfiguration(REGISTRY_CONF);
+    private static String ENV_VALUE;
+    static {
+        Map<String, String> map = System.getenv();
+        String env = map.get(ENV_SYSTEM_KEY);
```

Comment:
You can use System.getEnv(ENV_SYSTEM_KEY);

Reply:
OK